protocols/util/generate_mock_objects.py

```diff
@@ -940,7 +940,7 @@ def get_valid_clinical_report_cancer_4_2_0_SNAPSHOT():
 def get_valid_called_genotype_2_1_0():
     object_type = reports_2_1_0.CalledGenotype
     new_cg = MockModelObject(object_type=object_type).get_valid_empty_object()
-    new_cg.genotype = ''
+    new_cg.genotype = 'mock genotype'
```

Comment:
@greglever should we use as mocked genotype something like in the description `Zygosity as per VCF spec possibility phased such as 0|1, 1|0 for phased or 0/1 for unphased`. 0/1 should be sensible.

Reply:
This isn't intended to give valid data, just valid data types.

Reply:
I think the FactoryBoy() stuff @Antonior26 was working on is intended to give valid data.

Reply:
But we can change it if you like. 
What would you like ? `new_cg.genotype = '0/1'` ?